src/main/java/com/aquaticinformatics/aquarius/sdk/samples/create_service_models.sh

```diff
@@ -18,12 +18,15 @@ show_usage() {
 	exit_abort "$@"
 }
 
-Generator=../../../../../../../../../aquarius-sdk-net/src/SamplesServiceModelGenerator/bin/Release/net6.0/SamplesServiceModelGenerator.dll
+GeneratorProjectPath=../../../../../../../../../aquarius-sdk-net/src/SamplesServiceModelGenerator/SamplesServiceModelGenerator.csproj
+Generator=../../../../../../../../../aquarius-sdk-net/src/SamplesServiceModelGenerator/bin/Release/net8.0/SamplesServiceModelGenerator.dll
 
 ServerName=$1
 OutputPath=$2
 
-[ -f "$Generator" ] || exit_abort "Can't find $Generator. You'll need to build it first."
+command -v dotnet >/dev/null 2>&1 || exit_abort "This script requires the .NET 8 runtime. Grab it from here: https://www.microsoft.com/net/download"
+[-f "$GeneratorProjectPath"] || exit_abort "Can't find aquarius-sdk-net repo cloned to the same directory as aquarius-sdk-java"
```

Comment:
i'm getting a runtime issue with this locally. i'm not familiar with bash syntax, but it seems to require spaces before the `-f` and probably after ` `"` like the other lines